class ThrowCF41
{
	public static void main(String[] args)
	{
		System.out.println("In main");
		
		throw new ArithmeticException();

		//System.out.println("end of main"); CE: u r s
	}
}